okio/src/main/kotlin/okio/BufferedSource.kt

```diff
@@ -15,4 +15,82 @@
  */
 package okio
 
-expect interface BufferedSource : Source
+expect interface BufferedSource : Source {
+  val buffer: Buffer
+
+  fun exhausted(): Boolean
+
+  fun require(byteCount: Long)
+
+  fun request(byteCount: Long): Boolean
+
+  fun readByte(): Byte
+
+  fun readShort(): Short
+
+  fun readShortLe(): Short
+
+  fun readInt(): Int
+
+  fun readIntLe(): Int
+
+  fun readLong(): Long
+
+  fun readLongLe(): Long
+
+  fun readDecimalLong(): Long
+
+  fun readHexadecimalUnsignedLong(): Long
+
+  fun skip(byteCount: Long)
+
+  fun readByteString(): ByteString
+
+  fun readByteString(byteCount: Long): ByteString
```

Comment:
Lots of cases where default argument values with `@JvmOverloads` should be used here in common. Also in BufferedSink.

Reply:
Can address in more targeted PRs

Reply:
File an issue so we don't lose track? I wouldn't want to ship a version like this